### ISSUE TITLE
tracker: Fedora 38 Test Week on March 28 

### ISSUE THREAD

**Issue by sumantro93** [coreos/fedora-coreos-tracker#1440]
Hey Folks!

We are in beta now and this is a good time to start planning for a Test Week. 
I see new packages being added this time, crun-wasm being one. 
Do we intend to test something specifically that has landed new?
I will be writing more test cases if need be.

## Initial Tasks (to be done at least one week before test week)

- [x] Open this ticket in the `fedora-coreos-tracker` repo
- [x] Open a ticket with [Fedora QA](https://pagure.io/fedora-qa/issues).
  - [F36 example](https://pagure.io/fedora-qa/issue/695)

To be done after the Fedora QA folks have taken action on the QA ticket:

- [x] Confirm the Test Day page is created on the Fedora Wiki.
  - For example: <https://fedoraproject.org/wiki/Test_Day:Fedora_36_CoreOS_2022-04-04>
- [x] Confirm the Test Day results app is created.
  - For example: <https://testdays.fedoraproject.org/events/131>
- [x] Choose a day during the Test Week to host a video session for live debug help
- [x] Setup a Google Meet or other video conference session
- [x] Create a HackMD doc for capturing notes during live video session
- [x] Find volunteers to enumerate new documentation + test cases required for Test Week
  - Best done via dedicated video session
- [x] File an issue on `fedora-coreos-tracker` with TODO items.
  - For example: <https://github.com/coreos/fedora-coreos-tracker/issues/1147>
- [x] File a ticket requesting a Fedora badge is created
  - For example: <https://pagure.io/fedora-badges/issue/871>

## Announcing Test Week

Should be completed after the Initial Tasks are done

- [x] Draft an email to <coreos@lists.fedoraproject.org> announcing the Test Week
  - [x] Include a link to the Fedora Wiki
  - [x] Include a link to the Test Day results app
  - [x] Include a link to `fedora-coreos-tracker` for Test Week
  - [x] Include a link to the video conference
  - [x] Include a link to the HackMD doc
- [ ] Cross-post announcement email to discussion.fedoraproject.org with `#coreos` tag

- Example format: <https://hackmd.io/lCrVoW_RSMCRf2neiGhiPg>

## During Test Week

- Monitor `fedora-coreos-tracker` for new issues reported as part of Test Week
- Monitor #fedora-coreos on IRC for new issues reported as part of Test Week
- Ensure there is one or more representatives of Fedora CoreOS team present for live video session

## After Test Week

- [x] Update `fedora-coreos-tracker` ticket with any issues found
- [x] Update `fedora-coreos-tracker` ticket with any documentation updates made
- [x] Review Test Day results app and follow-up on any errors reported, if possible
- [ ] Follow up with the Fedora Badges ticket with Fedora Account System (FAS) usernames that participated in Test Week

**Comment by dustymabe** [comment]
Fedora QA ticket for F38 CoreOS Testing: https://pagure.io/fedora-qa/issue/721

**Comment by ravanelli** [comment]
I'm tagging here people that may want to work as volunteers to enumerate new documentation + test cases required for Test Week
@bgilbert @miabbott @mike-nguyen @davdunc @fifofonix @jasonbrooks @cgwalters

**Comment by sumantro93** [comment]
event : https://testdays.fedoraproject.org/events/153

**Comment by ravanelli** [comment]
New tests cases:

* Raspberry Pi 4 test case: https://fedoraproject.org/wiki/QA:Testcase_CoreOS_Raspberry_Pi
* Systemd unit service : https://fedoraproject.org/wiki/QA:Testcase_FCOS_systemd_unit_service
* Autologin TC https://fedoraproject.org/wiki/QA:Testcase_FCOS_autologin_setting_custom_hostname
* nmstate: https://fedoraproject.org/wiki/QA:Testcase_nmstate 

No issues were found during the tests

**Comment by dustymabe** [comment]
Looks like this is pretty much all done, just waiting on someone to act on the badges ticket request, which is a separate ticket.

I'll close this out. Thanks @ravanelli for organizing this!
